### ISSUE TITLE
42139082: Assertion in _UIDragSnappingFeedbackGenerator.userInteractionStarted at UIDragSession start

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#20083]
#### Description

Summary:
We're testing Drag & Drop capabilities in our application, and we've noticed a rare crash that's not easily reproducible:

Here's the crash log:

```
*** Assertion failure in -[_UIDragSnappingFeedbackGenerator userInteractionStarted], /BuildRoot/Library/Caches/com.apple.xbs/Sources/UIKitCore/UIKit-3698.79.2/_UIDragFeedbackGenerator.m:175
2018-07-12 14:42:51.323435-0500 Viewer[10264:809379] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: '<_UIDragSnappingFeedbackGenerator: 0x2811dcfa0: prepared=1> is already being interacted with'
```

We've seen this crash occur when the drag "lift" animation is just completed on a UICollectionView, and the user starts dragging. There are no other factors that we've identified as related to the crash occurring.

Steps to Reproduce:
Start a drag session by long pressing an item in a collection view.

Expected Results:
The drag session is started correctly, and no crashes are experienced.

Actual Results:
Sometimes, with no clear reason, an exception is thrown from _UIDragSnappingFeedbackGenerator.userInteractionStarted and the app is terminated.

Version:
12 beta 3

Notes:

-
Product Version: 12 beta 3
Created: 2018-07-12T21:59:40.452060
Originated: 2018-07-12T16:59:00
Open Radar Link: http://www.openradar.me/42139082

**Comment by openradar-mirror** [comment]
Resolved: Duplicate of 41387612
Modified: 2018-07-23T20:49:05.107060

**Comment by openradar-mirror** [comment]
Resolved: Duplicate of 41387612
Modified: 2018-07-23T20:49:05.107060
